src/formatter.py

```diff
@@ -5,11 +5,26 @@
 cfr: https://github.com/sagacify/logger
 """
 import datetime
+import json
 import os
 import socket
 import sys
 import time
-import json
+import traceback
+
+
+def error_serializer(error):
+    """Serialize error as bunyan.
+
+    When in doubt, ask what would bunyan do?
+    https://github.com/trentm/node-bunyan/blob/master/lib/bunyan.js#L1141
+    """
+    trace = traceback.format_stack(error.__traceback__)
+    return {
+        'message': ' '.join(error.args),
+        'name': error.__class__.__name__,
+        'stack': [line for line in trace]
+    }
 
 
 def extra_serializer(obj):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 5 returns or yields, which is more than the 3 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1142)